ospsuite-r-specifics/collaboration_guide.md

```diff
@@ -0,0 +1,58 @@
+# R Development Collaboration Guide
+
+## Rules
+
+The following set of rules should be followed by all contributors and checked by
+all reviewers of the OSPS R projects.
+
+1. Each change made to the codebase should have a clear purpose and exist in an
+isolated context
+  1.1 Each change should be related to an issue created on the project's repository
+  1.2 Each change should be made in a separate branch
+  1.3 Each change should be proposed through a pull request
+2. Each change or addition to the code should be documented and controlled
+  2.1 Each change in the code should be reflected in the documentation. If 
+  documentation is not present, then it should be created.
+  2.2 Each change should be tested. If new cases emerge, then they should be 
+  tested, even if some tests are already present. If no tests are present, then
+  they should be created.
+3. Each change should be easily understandable and accessible
+  3.1 Each change should have a corresponding entry in the `NEWS` file.
+4. Each change must respect the coding style of the project.
+  4.1 Each change must be processed by the `{styler}` package before being proposed
+5. Each change should be reviewed before being merged
+  5.1 Each Pull Request should be reviewed by at least one other contributor
+  5.2 Pull requests that are not ready should be marked as draft until they are ready for review
+
+
+## Recommended Workflows
+
+### Changing Code
+
+The recommended workflow for contributing to the OSPS R projects heavily relies on the [`{usethis}`](https://usethis.r-lib.org) package and it family of function [`pr_*()`](https://usethis.r-lib.org/articles/pr-functions.html).
+
+#### Prerequisites
+
+This workflow imply that:
+
+-   R and R Studio are installed,
+-   A GitHub account is available and setup to work within RStudio,
+-   The `{usethis}` package is installed,
+-   A clone (from original repository or from a fork) has been created.
```

Comment:
A local clone of the repository?

It took me a while to understand.

Reply:
How would you phrase this ?

Clone is the "official" term for getting source code on computer with a link the the remote repository so I don't really have an alternative to propose.

Reply:
Add "local? :D

Reply:
done